core/arch/arm/kernel/tee_time_arm_cntpct.c

```diff
@@ -54,3 +56,40 @@ static const struct time_source arm_cntpct_time_source = {
 };
 
 REGISTER_TIME_SOURCE(arm_cntpct_time_source)
+
+/*
+ * We collect jitter using cntpct in 32- or 64-bit
+ * mode that is clocked at the cpu core rate.
```

Comment:
Comment still needs fixing.
In addition, abbreviations should be upper case (`s/cpu/CPU/` and `s/cntpct/CNTPCT/`).